source/puppet/5.3/release_notes_agent.md

```diff
@@ -29,6 +32,16 @@ The `puppet-agent` package installs the latest version of Puppet 5.
 
 Also of interest: [About Agent](./about_agent.html), and the [Puppet 5.2.0][], [Puppet 5.1.0][], and [Puppet 5.0.0][] release notes.
 
+## Puppet agent 5.3.3
+
+Released November 6, 2017.
+
+This release of Puppet Platform contains several Puppet and Facter bug fixes.
+
+### Component updates
+
+This release includes component updates to [Puppet 5.3.3][], [Facter 3.9.3][], [MCollective 2.11.4][], pcp-client 1.5.5, and [pxp-agent][] 1.8.1.
```

Comment:
FYI, vanagon still shows pcp-client 1.5.4 and pxp-agent 1.8.0.

Reply:
From what I can tell it doesn't *look* like either of those have releases but I'm pretty sure @MikaelSmith would be the one who can confirm whether or not those two have version bumps.

Reply:
I think @branan made the final decision on that. My understanding is - because there were only test changes - we decided not to include version bumps for pcp-client and pxp-agent.

Reply:
Right. The version bump tickets were created, but we closed them `wontfix` because there were no code changes. We need a better workflow around when something has commits vs. when there are actual code changes that we need to ship. Turns out that's actually not easy to automate :(